agenta-web/src/components/EvaluationTable/SimilarityMatchEvaluationTable.tsx

```diff
@@ -324,6 +327,36 @@ const SimilarityMatchEvaluationTable: React.FC<SimilarityMatchEvaluationTablePro
                 )
             },
         },
+        {
+            title: "Similarity",
+            dataIndex: "similarity",
+            key: "similarity",
+            width: 200,
+            align: "center" as "left" | "right" | "center",
+            render: (text: any, record: any, rowIndex: number) => {
+                let tagColor = ""
+                if (record.score === "true") {
+                    tagColor = "green"
+                } else if (record.score === "false") {
+                    tagColor = "red"
+                }
+
+                const similarity = rows[rowIndex].similarity
+                return (
+                    <Spin spinning={similarity ? false : true}>
+                        <Space>
+                            <div>
+                                {similarity != undefined && (
```

Comment:
please perform a type-sensitive check (!==). Right now even similarity = 0 will resolve this condition to true.